pyproject.toml

```diff
@@ -111,7 +112,18 @@ lint.ignore = [
     "N999",
     "UP007"
 ]
-exclude = ["mapclassify/tests/*", "docs/*"]
+extend-include = [
+    "docs/conf.py"
+]
+
+[tool.ruff.lint.per-file-ignores]
+ "*tests/test_*.py" = [
+    "A004",  # Import is shadowing a Python builtin
```

Comment:
What's triggering this?

Reply:
`bin`

Reply:
$\rightarrow$ [`classifiers.bin()`](https://github.com/pysal/mapclassify/blob/72a4e7e2177104ecc4cc876f75641ec9f479713d/mapclassify/classifiers.py#L341)
$\rightarrow$ imported in [`test_mapclassify.py`](https://github.com/pysal/mapclassify/blob/72a4e7e2177104ecc4cc876f75641ec9f479713d/mapclassify/tests/test_mapclassify.py#L7)